blog/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin
```

Comment:
Add blank line between different import groups to follow PEP-8 here and in the other places as well